build_tools/benchmarks/common/linux_device_utils.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Utils for accessing Linux device information."""
+
+import re
+from typing import Sequence
+
+from .benchmark_definition import (execute_cmd_and_get_output, DeviceInfo,
+                                   PlatformType)
+
+
+def _get_lscpu_field(field_name: str, verbose: bool = False) -> str:
+  output = execute_cmd_and_get_output(["lscpu"], verbose)
+  (value,) = re.findall(f"^{field_name}:\s*(.+)", output, re.MULTILINE)
+  return value
+
+
+def get_linux_cpu_arch(verbose: bool = False) -> str:
```

Comment:
Please add docstrings for these functions, which should also include an example so that it's easier to understand what to expect for arch/features/models, etc.

Reply:
Done.